web-src/src/services/FirefallService.js

```diff
@@ -51,6 +54,7 @@ export class FirefallService {
       .post({
         queryId,
         sentiment,
+        isInternal: this.isInternal,
```

Comment:
@askayastha22 If the server-side check depends on what's received from the frontend, would it be easy to tamper with?

Reply:
@vtsaplin I tried hacking `isInternal` by adding a breakpoint in ShellProvider.js and changing the value of `isInternal` in order to make unauthorized requests to Firefall. The frontend got disabled when I did this. So I am assuming the hacker would never be able to click the `Generate` button.

Reply:
@vtsaplin Let me verify this.

Reply:
@vtsaplin Yes, `isInternal` can be tampered. I was trying very hard to avoid sending `isInternal` to the backend. Unfortunately, Unified Shell's feature flag API (via their proxy APIs) has a very opinionated way of storing the IMS Org ID as the key of the user context, which will override any valid `isInternal` values for user context instances sent from the frontend. In other words, backend always has to know the value of `isInternal` somehow to send as context to LD.

There are two options unless you have some ideas:
1. Just accept `isInternal` can be tampered with.
2. Use LD client-side SDK.
3. May be you have some ideas?